openedx/core/djangoapps/user_api/accounts/tests/test_image_helpers.py

```diff
@@ -25,17 +26,17 @@ def setUp(self):
         self.user = UserFactory()
         # Ensure that parental controls don't apply to this user
```

Comment:
Can this be removed now that auto auth will give us a test user who does not require parental consent?